src/main/java/com/xiaohengjin/Pojo2ProtoCore.java

```diff
@@ -47,7 +47,15 @@ public void start() {
         CopyPasteManager.getInstance().setContents(new StringSelection(protoContent));
     }
 
-    public void go(PsiClass clazz) {
+    private void go(PsiClass clazz) {
+        if(clazz.isEnum()) {
```

Comment:
code format: if and () should have a blank

Reply:
Done.